### DISCUSSION TITLE
Restore Factory Firmware Flysky I6X (Fixed Screen Issue)

### DISCUSSION THREAD

**Discussion by ysabxe** [OpenI6X/opentx#385] [Show and tell]
**To restore original firmware after installing OpenTX:**

1. Enter **DFU Mode** by holding 2 buttons and Turn ON controller (or R53)
![4](https://github.com/OpenI6X/opentx/assets/155847328/6f7f8b0a-9722-491f-bcfa-c1d3a3ad774c)
3. Install **DFUProgrammer_English.msi and DFUDriver**
4. Flash **Stocki6x.bin via DFUProgrammer** (USB - Connect, Chose bin - Update)
5. Enter **Firmware Update Mode by holding 2 buttons and Turn ON controller**
![UpgradeMode](https://github.com/OpenI6X/opentx/assets/155847328/a4487d69-2373-42a8-803a-15a560262bce)
6. Run **UpdateFirmwareFS-i6X.exe and hit Update**

**Files**
[RestoreFirmwareFlyskyI6X.zip](https://github.com/OpenI6X/opentx/files/13851367/RestoreFirmwareFlyskyI6X.zip)

**Comment by ajjjjjjjj** [comment]
Hi, what is the "Fixed Screen Issue" you are referring to?

**Reply by ysabxe** [reply]
After restoring stock .bin file it was impossible to use controller (screen with black noise). That is the proper way
